### PR TITLE
Assignment 2. Version route to plugin.

### PR DESCRIPTION
**Pull request by anthonyringoet**

### CODE REVIEW

**Comment by AdriVanHoudt** [comment]
Look at #43 for general feedback!

**Comment by anthonyringoet** [comment]
Thanks for the heads up @AdriVanHoudt 
Updates pushed.

File: lib/index.js [hunk 1]
```diff
@@ -15,23 +15,18 @@ internals.init = function () {
...
-        method: 'GET',
-        path: '/version',
-        config: {
-            description: 'Returns the version of the server',
-            handler: function (request, reply) {
-
-                return reply({ version: Package.version });
-            }
+    server.register(Version, function (err) {
+        if (err) {
```
Comment (AdriVanHoudt) [inline lib/index.js]: Use the same error handling here as in `server.start()`

File: lib/version.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        config: {
+            description: 'Returns the version of the server',
+            handler: function (request, reply) {
+
+                return reply({ version: Package.version });
+            }
+        }
+    });
+
+    next();
```
Comment (AdriVanHoudt) [inline lib/version.js]: use `return next()` here for readability

File: lib/version.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+exports.register.attributes = {
+    name: 'version'
+};
```
Comment (AdriVanHoudt) [inline lib/version.js]: newline at end of file

**Pull request closed without merging**